samcli/lib/test_runner/invoke_testsuite.py

```diff
@@ -0,0 +1,116 @@
+"""
+Kicks off a testsuite in a Fargate container
+"""
+from typing import List
+import logging
+from samcli.commands.exceptions import ReservedEnvironmentVariableException
+from samcli.lib.utils.colors import Colored
+
+LOG = logging.getLogger(__name__)
+
+
+class SuiteRunner:
```

Comment:
If this `SuiteRunner` is specific to Fargate, I would indicate that in the class name here.